src/stylesheets/app.scss

```diff
@@ -165,4 +179,500 @@ nav {
   width: 1px;
 }
 
+// home/index page
+.home-container {
+  margin: 0 auto;
+  width: rd(950, 1440);
+
+  @include breakpoint(max-width 900px) {
+    width: rd(1200, 1440);
+  }
+  @include breakpoint(max-width 900px) {
+    width: rd(1300, 1440);
+  }
+  .title {
+    display: block;
+    margin-bottom: rem(40);
+    img {
+      display: block;
+      margin: 0 auto;
+      max-width: 100%;
+      padding: .5rem;
+    }
+  }
+  header {
+    @extend %clearfix;
+  }
+}
+.category-selector {
+  float: right;
+  margin-bottom: rem(36);
+  label {
+    margin-right: rem(11);
+  }
+}
+// author bio page
+.authorBio--container {
```

Comment:
Should this be `authorBio--container` or `authorBio__container`?  It looks like a child of `authorBio` to me, not an alternate version.

Also, `kabob-case` classes please, no `camelCase`